src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -127,6 +130,8 @@ public function __construct(?string $name, ?string $dataClass, EventDispatcherIn
         $this->dataClass = $dataClass;
         $this->dispatcher = $dispatcher;
         $this->options = $options;
+
+        $this->isEmptyCallback = [__CLASS__, 'isEmpty'];
```

Comment:
What about using an anonymous function here instead?

Like this:

```php
$this->isEmptyCallback = static function ($modelData): bool {
    return FormUtil::isEmpty($modelData) ||
            // arrays, countables
            ((\is_array($modelData) || $modelData instanceof \Countable) && 0 === \count($modelData)) ||
            // traversables that are not countable
            ($modelData instanceof \Traversable && 0 === iterator_count($modelData));
};
```

Reply:
The advantage of having a dedicated method would be to reuse it (in 4.4) in `Form::isEmpy()` when the new `FormConfigInterface::getIsEmptyCallback()` method is not yet implemented by custom `FormConfigInterface` implementations.

Reply:
Who would reuse it? I don't see any place in the Form component doing that and since the method would be internal, I don't see it makes much sense in anyone else reusing it.

Reply:
Because of https://github.com/symfony/symfony/pull/32747#pullrequestreview-267126592.
In `Form::isEmpty()` I will need to add : 
```